src/scrape_up/codeforces/user.py

```diff
@@ -7,7 +7,7 @@ class USER:
     Fetches user data from CodeForces Competitive Platform
     """
 
-    user_url = "https://codeforces.com/profile/{}"
+    user_url = "https://codeforces.com/api/user.info?handles={}"
```

Comment:
@metflekx | As per the motive of the project, we are building an API alternative so that we do not have to face issues that we usually face while working using an API like rate limiting, authentication and authorization, etc. Here you are replacing the URL with the API endpoint, this doesn't align with the project motive, right? 

Hope you get my point.